CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Changes since v2.38.1
   - When creating attachment license, the save button no longer activates until all attachments have been successfully uploaded. (#3292)
   - Attachment upload error is indicated by small icon next to the upload button (detailed error is still shown in top of the page).
   - Pending attachment upload is indicated by spinner next to the upload button.
+- Edit buttons on the administration pages are hidden when the user is not permitted to edit
```

Comment:
I would like to change this a little bit to "administration detail pages" and "like they are hidden on the list pages already". Just to emphasize that this is just about the same functionality now in another place. Also we should include the issue number here.